src/dump.rs

```diff
@@ -256,8 +275,8 @@ pub struct DumpClassFunction {
 pub struct DumpClassEvent {
     pub name: String,
 
-    #[serde(default = "Vec::new")]
-    pub tags: Vec<String>,
+    #[serde(default = "BTreeSet::new")]

```

Comment:
```suggestion
    #[serde(default)]
```

and here... I hope using GitHub suggestions makes this easy enough :)